src/twitter.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "twitter-v2" {
+    export interface Credentials {
+        consumer_key?: string;
+        consumer_secret?: string;
+        bearer_token?: string;
+        access_token_key?: string;
+        access_token_secret?: string;
+    }
```

Comment:
We should probably define `Credentials` as a union type (which is how it's used internally)

```typescript
interface ApplicationCredentials {
  consumer_key: string;
  consumer_secret: string;
  bearer_token?: string;
}

interface UserCredentials {
  consumer_key: string;
  consumer_secret: string;
  access_token_key: string;
  access_token_secret: string;
}

interface Credentials = ApplicationCredentials | UserCredentials;
```